lib/topological_inventory/mock_source/entity/container_node.rb

```diff
@@ -12,7 +12,7 @@ def to_hash
 
       def references_hash
         {
-          :cross_link_vms => {:uid_ems => nil} # TODO: link_to(:vms)
+          # :cross_link_vms => {:uid_ems => nil} # TODO: link_to(:vms)
```

Comment:
Does this need to be commented out vs just removed?  If we need to keep the `#TODO` around  I don't mind that.

Reply:
I'll remove it at all, purpose of this collector isn't in covering all non-standard relations